pipelines/ui5-sap-cp/Jenkinsfile

```diff
@@ -21,15 +23,21 @@ node() {
 
   stage("Clone sources and setup environment"){
     deleteDir()
+    Map neoDeployConfiguration, mtaBuildConfiguration
     dir(APP_PATH) {
       checkout scm
       setupCommonPipelineEnvironment script: this, configFile: CONFIG_FILE
+      prepareDefaultValues script: this
+      neoDeployConfiguration = ConfigurationMerger.merge(ConfigurationLoader.stepConfiguration(this, 'neoDeploy'), ['host', 'account', 'credentialsId'],
+                                                         ConfigurationLoader.defaultStepConfiguration(this, 'neoDeploy'))
+      mtaBuildConfiguration = ConfigurationMerger.merge(ConfigurationLoader.stepConfiguration(this, 'mtaBuild'), ['mtaHome'],
+                                                         ConfigurationLoader.defaultStepConfiguration(this, 'mtaBuild'))
     }
-    MTA_JAR_LOCATION = commonPipelineEnvironment.getConfigProperty('MTA_HOME')
-    NEO_HOME = commonPipelineEnvironment.getConfigProperty('NEO_HOME')
-    DEPLOY_HOST = commonPipelineEnvironment.getConfigProperty('DEPLOY_HOST')
-    CI_DEPLOY_ACCOUNT = commonPipelineEnvironment.getConfigProperty('CI_DEPLOY_ACCOUNT')
-    neoCredentialsId = commonPipelineEnvironment.getConfigProperty('neoCredentialsId')
+    MTA_JAR_LOCATION = mtaBuildConfiguration.mtaHome ?: commonPipelineEnvironment.getConfigProperty('MTA_HOME')
+    NEO_HOME = neoDeployConfiguration.neoHome ?: commonPipelineEnvironment.getConfigProperty('NEO_HOME')
+    DEPLOY_HOST = neoDeployConfiguration.host ?: commonPipelineEnvironment.getConfigProperty('DEPLOY_HOST')
+    CI_DEPLOY_ACCOUNT = neoDeployConfiguration.account ?: commonPipelineEnvironment.getConfigProperty('CI_DEPLOY_ACCOUNT')
+    neoCredentialsId = neoDeployConfiguration.credentialsId ?: commonPipelineEnvironment.getConfigProperty('neoCredentialsId')
```

Comment:
And all this properties are also not needed.

Reply:
And all the code related to this members can be removed.

Reply:
At the moment our approach is to validated these properties. We know in the meantime that our validation does not work eg. when executing steps in dockers or on Jenkins slave nodes. But there is no decision to remove the current approach of a validation. As long as there is no such decision we validate. By the way: I personally vote for removing the validation.